dradis/dradis-ce
# Add issue tags in API json builder and update specs

### Spec
Currently, there are no tags when fetching issues from the API.

**Proposed Solution**
Add tags in the issue json builder.

### How to test:
1. Add an issue in a project
2. Use the API endpoint to fetch the issue
3. Confirm that the issue tags was returned
